splunk-otel-android/src/main/java/com/splunk/rum/BandwidthTracker.java

```diff
@@ -0,0 +1,61 @@
+package com.splunk.rum;
+
+import java.time.Clock;
+import java.util.ArrayList;
+import java.util.List;
+
+import kotlin.collections.ArrayDeque;
```

Comment:
Oh it's Kotlin's `ArrayDeque`... I thought it was a bit strange, since the Java one does not implement `List`

Reply:
Oh yeah, I have a weakness about never looking at imports. 🙈 
Changing to the java version is nice as you point out below -- will do.